lib/OpenQA/WebAPI.pm

```diff
@@ -172,8 +172,9 @@ sub startup {
 
     # Favicon
     $r->get('/favicon.ico' => sub { my $c = shift; $c->render_static('favicon.ico') });
-    $r->get('/index'       => [format => ['html', 'json']])->to('main#index');
```

Comment:
would be great if the .json way can be preserved by automatically loading all dynamic content if the json route is triggered. Alternatively – as you stated that "dashboard_build_results" includes this information – then you could redirect there on .json as finding that sub-page would be hard otherwise.

Reply:
I personally think the effort of implementing any of these options and the resulting additional complexity of the code is not worth it. The `.json` route is a nice bonus which is still preserved but just at another place.

(Not sure how easy it is within Mojolicious to have different code paths for different formats. So far this worked merely by using `respond_to` in the end.)

Reply:
I see. Maybe you can just try out how https://github.com/os-autoinst/openqa_review behaves?

Reply:
Just by looking at the code it seems the script would be unaffected because it only queries the build results from the individual job group pages: https://github.com/os-autoinst/openqa_review/blob/master/openqa_review/openqa_review.py#L441

Reply:
Yes, sounds good. Let's hope no one else is using that then :)